modules/react/button/lib/BaseButton.tsx

```diff
@@ -21,12 +21,14 @@ import {CSSObject} from '@emotion/styled';
 import {CanvasSystemIcon} from '@workday/design-assets-types';
 
 export interface ButtonContainerProps extends BoxProps, GrowthBehavior {
+  /**
+   * Override default colors of a button. The default will depend on the button type
+   */
+  // default marked as undefined so the doc system doesn't render out the the button colors type
```

Comment:
No longer need this comment since the `@default {undefined}` was removed for `colors` [here](https://github.com/Workday/canvas-kit/pull/1962/commits/ee1344065b458ed054f8404468b7d6e2ce2b2581#diff-f49674572a92e7ec647c2aab62fe41ff3386e81037b1a05a3fc8e02c9930aa11).

```suggestion
```